phys2bids/phys2bids.py

```diff
@@ -109,26 +111,44 @@ def print_json(outfile, samp_freq, time_offset, ch_name):
         File containing information for BIDS.
     """
     start_time = -time_offset
-    summary = dict(SamplingFrequency=samp_freq,
-                   StartTime=round(start_time, 4),
-                   Columns=ch_name)
+    summary = dict(
```

Comment:
I dont' get why this is not formatted as long list - can someone explain it to me? (I would force the long list adding a comma after `Columns=ch_name`).

Reply:
It's not long enough to be formatted that way default, but I'll add the trailing comma to do it.